services-camel-app/src/test/java/edu/unc/lib/boxc/services/camel/images/ImageEnhancementsRouterTest.java

```diff
@@ -89,7 +86,7 @@ public void testAccessCopyRouteNoForceNoFileExists() throws Exception {
         createContext(accessCopyRoute);
 
         var jp2Endpoint = getMockEndpoint("mock:bean:jp2Processor");
```

Comment:
this doesn't exist anymore since we're calling `.bean(jp2Processor)` directly, which is why its always being called 0 times. But you should be able to do `verify(jp2Processor).process(any(Exchange.class));` instead to check if its been called or not